mrSerious/StackOverflow-lite
# #160359011 Build front-end to consume the post answer endpoint

#### What does this PR do?
Have a front-end to consume the update answer end-point
#### Description of Task to be completed?
Have a working front-end to consume the following endpoint.
`PUT /api/v1/questions/:questionId/answers/:answerId`
#### How should this be manually tested?
After cloning the repo, cd into the folder and run `npm install`
Check that front-end consumes the endpoint as follows:
- Open your browser
- Navigate to the page to view a questions details.
- Click the edit link on an answer.
- Complete and submit the form to edit an answer.
- Your answer should be updated successfully.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#160359011](https://www.pivotaltracker.com/n/projects/2189147/stories/160359011)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
